src/Components/Alert/Components/CriteriaPills.tsx

```diff
@@ -103,17 +54,12 @@ const CriteriaPills: FC<CriteriaPillsProps> = ({
   )
 }
 
-export const CriteriaPillsFragmentContainer = createFragmentContainer(
-  CriteriaPills,
-  {
-    previewSavedSearch: graphql`
-      fragment CriteriaPills_previewSavedSearch on PreviewSavedSearch {
-        labels {
-          displayValue
-          field
-          value
-        }
-      }
-    `,
-  }
-)
+const CriteriaPillsPlaceholder: FC = () => {
+  return (
+    <>
+      <SkeletonBox width={150} height={30} />
+      <SkeletonBox width={80} height={30} />
+      <SkeletonBox width={100} height={30} />
+    </>
```

Comment:
That could work as well 👍 We would just need to get the correct width somehow (`150`, `80`, `100` or random).

Reply:
With random, or the same value for all components

Reply:
Hmm, it would actually be longer because we have a character limit per line:

```
    <>
      {[...Array(3)].map((_, index) => (
        <SkeletonBox
          width={Math.floor(Math.random() * (150 - 80 + 1)) + 80}
          height={30}
          key={index}
        />
      ))}
    </>
```